docs/using.md

```diff
@@ -164,16 +174,23 @@ steps:
         value: "world"
 ```
 
-### Resource shared between tasks
+##### Configure Entrypoint image
+
+To run a step needs to pull an `Entrypoint` image. Maybe the image is hard to
```

Comment:
I think something is missing in the first sentence

Reply:
lol whoops :sweat_smile: